docs/tutorials/api/clusters.rst

```diff
@@ -0,0 +1,222 @@
+Clusters
+========
+
+.. raw:: html
+
+    <p><a href="https://colab.research.google.com/github/run-house/runhouse/blob/stable/docs/notebooks/api/clusters.ipynb">
+    <img height="20px" width="117px" src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a></p>
+
+A cluster is the most basic form of compute in Runhouse, largely
+representing a group of instances or VMs connected with Ray. They
+largely fall in two categories:
+
+1. Static Clusters: Any machine you have SSH access to, set up with IP
+   addresses and SSH credentials.
+2. On-Demand Clusters: Any cloud instance spun up automatically for you
+   with your cloud credentials.
+
+Runhouse provides various APIs for interacting with remote clusters,
+such as terminating an on-demand cloud cluster or running remote CLI or
+Python commands from your local dev environment.
+
+Let’s start with a simple example using AWS. After making sure your
+``~/.aws/credentials`` file is set up with access to create instances in
+EC2, you can install Runhouse and create on-demand clusters in AWS.
+
+.. code:: ipython3
+
+    ! pip install "runhouse[aws]"
+
+On-Demand Clusters
+------------------
+
+We can start by using the ``rh.cluster`` factory function to create our
+cluster. By specifying an ``instance_type``, Runhouse sets up an
+On-Demand Cluster in AWS EC2 for us.
+
+Each cluster must be provided with a unique ``name`` identifier during
+construction. This ``name`` parameter is used for saving down or loading
+previous saved clusters, and also used for various CLI commands for the
+cluster.
+
+.. code:: ipython3
+
+    import runhouse as rh
+
+    aws_cluster = rh.cluster(name="test-cluster", instance_type="CPU:2")
```

Comment:
think adding provider="aws" will be more explicit/robust if anyone's rerunning the tutorial and may have another provider set up.

```suggestion
    aws_cluster = rh.cluster(name="test-cluster", instance_type="CPU:2", provider="aws")
```